Tools/AP_Bootloader/board_types.txt

```diff
@@ -222,6 +222,7 @@ AP_HW_SPRACINGH7RF                   1108
 AP_HW_AEROFOX_GNSS_F9P               1109
 AP_HW_JFB110                         1110
 AP_HW_SDMODELH7V1                    1111
+AP_HW_BLITZF7                        1112
```

Comment:
I suggest you reserve this ID in a separate PR - otherwise someone else might snatch it away and then you get to go and update all of your boards' bootloaders to match your new ID...

Reply:
Fixed